### ISSUE TITLE
[Suggestion] - Option to disable explosions

### ISSUE THREAD

**Issue by SSKirillSS** [Shinoow/AbyssalCraft#375]
#### Description:

I propose to add the ability to turn off all explosions (or turn them off separately) in AbyssalCraft configuration file. Most explosions cause too much damage to the map, especially if you play on the server.
![image](https://user-images.githubusercontent.com/38190325/60450458-94c8db80-9c32-11e9-98ca-c47b0f3ad619.png)

#### Affected versions:
- Minecraft: 1.12.2
- Forge: 14.23.5.2836
- AbyssalCraft: AbyssalCraft-1.12.2-1.9.8

**Comment by Shinoow** [comment]
Judging by the image, maybe you shouldn't set off the explosive if you don't want a crater...
It would be pretty anti-climactic if someone set off a bomb whose tooltip contains the words "heavy explosive", then nothing happens.
On a server, I'm pretty sure there's a plugin (or mod, if you're not using Sponge, Bukkit, or something else besides Forge) where you can assign permissions to restrict usage of something (or, if you have CraftTweaker, remove the recipe). In the past I've read of servers banning items/blocks, so there's likely something in place stopping the item/block from being used.

For summoning Sacthoth, you don't even need to set the bomb off. You can summon him through a ritual, or craft a spawn egg with the bomb.

**Comment by SSKirillSS** [comment]
Yes, I use Spongeforge. However, I could not ban the explosion (although perhaps I just tried too little).


In order not to search for a solution for too long, I simply turned off the explosion in the AbyssalCraft code.
![image](https://user-images.githubusercontent.com/38190325/60629563-e625c600-9dfe-11e9-9225-0e42e0692113.png)
![image](https://user-images.githubusercontent.com/38190325/60629568-e9b94d00-9dfe-11e9-976a-22e654f052e3.png)
